### PR TITLE
Add ability to shutdown

### PR DESCRIPTION
**Pull request by gazpachoking**
I'm not exactly sure this is the right way to shutdown, but it seems to work cleanly (the on_shutdown stuff gets called.) Also not sure if a GET to `/core/shutdown` is the right place to put this.

### CODE REVIEW

File: spritzle/resource/core.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#   51 Franklin Street, Fifth Floor
+#   Boston, MA    02110-1301, USA.
+#
+
+from aiohttp import web
+
+routes = web.RouteTableDef()
+
+
+@routes.get('/core/shutdown')
```
Comment (aresch) [inline spritzle/resource/core.py]: Should this be GET? I feel like it should be a POST or maybe DELETE /core?

File: spritzle/resource/core.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+async def get_shutdown(request):
+    def shutdown():
+        raise web.GracefulExit()
```
Comment (aresch) [inline spritzle/resource/core.py]: I assume this runs the app on_shutdown handler?

**Review by aresch — CHANGES_REQUESTED** [review]

File: spritzle/resource/core.py (hunk 1 above)
Comment (gazpachoking) [reply spritzle/resource/core.py]: Yeah, I wondered that too. I'm fine with either.

File: spritzle/resource/core.py (hunk 2 above)
Comment (gazpachoking) [reply spritzle/resource/core.py]: Yep.

File: spritzle/resource/core.py (hunk 1 above)
Comment (aresch) [reply spritzle/resource/core.py]: Ok I think DELETE /core makes the most sense.

**Comment by gazpachoking** [comment]
Should the function be named delete_core as well? That seems to fit with the standard, though it describes less what it actually does.

**Comment by aresch** [comment]
Yes, delete_core() seems like the best name.  We could always start adding doc strings to the functions.. in fact we should be updating interface.md as we change the interface.

**Pull request merged**